### DISCUSSION TITLE
Contributing code, guidance please

### DISCUSSION THREAD

**Discussion by weyert** [cypress-io/cypress#15681] [General]
I am interested to submit my code changes I made to my fork of Cypress which adds Webkit browsers support  that helped me testing a webkit engine based app in macOS and iOS (sorry not a Windows/Linux user). 

I am wondering what Cypress expectations are regarding submitting such code especially because it has a lot changes and implements a new protocol/driver and it won't support Safari builds.

**Comment by weyert** [comment]
Cypress, I  assume there is no interest?

**Reply by bahmutov** [reply]
There is interest of course, please take a look at our https://github.com/cypress-io/cypress/blob/develop/CONTRIBUTING.md guide

**Comment by flotwig** [comment]
@weyert If you want to take a look at the work I've done on #15533, I have the Playwright WebKit running and a CDP connection being established, but no other automation has been added on top of that. If you'd like to take the launching from that PR and combine it with the driving you've done that would be more than welcome, or you can simply submit what you have so far if it replaces my work.

**Reply by weyert** [reply]
@flotwig I have submitted my work in my pull request
